db/migrate/20160404132250_remove_katello_from_notification_name.rb

```diff
@@ -0,0 +1,26 @@
+class RemoveKatelloFromNotificationName < ActiveRecord::Migration
+  class FakeMailNotification < ActiveRecord::Base
+    self.table_name = 'mail_notifications'
+  end
+
+  def up
+    FakeMailNotification.all.each do |notification|
+      if notification_names.keys.include?(notification.name)
+        new_name = notification_names[notification.name]
+        FakeMailNotification.where(:name => new_name).destroy_all
```

Comment:
If the user has katello_promote_errata already it'll be assigned to users.  If they run seeds before migration they'll have katello_promote_errata and the new promote_errata, we need to delete that one.